Agenda-client/package.json

```diff
@@ -7,13 +7,18 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "test": "jest"
   },
   "dependencies": {
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
Is this needed as a dep? Shouldn't it be a dev dep?

Reply:
fixed